apps/examples/testcase/le_tc/network/tc_net_pbuf.c

```diff
@@ -118,7 +118,7 @@ static void tc_net_pbuf_layer_raw_tx_p(int fd)
 	args.type = PBUF_RAM;
 
 	ret = ioctl(fd, TESTIOC_NET_PBUF, &args);
-	TC_ASSERT_GEQ("pbuf",ret, 1);
+	TC_ASSERT_GEQ("pbuf", ret, 1);
```

Comment:
Please split a commit for fixing coding rule violations.

Reply:
ok.